R/sjImportSPSS.R

```diff
@@ -650,7 +656,10 @@ sji.getValueLabelValues <- function(x, sort.val = TRUE) {
   # nothing found? then leave...
   if (is.null(attr.string)) return(NULL)
   # get values
-  val.sort <- as.numeric(unname(attr(x, attr.string, exact = T)))
+  if (is.character(x))
+    values <- unname(attr(x, attr.string, exact = T))

```

Comment:
`values` should be replaced with `val.sort`, or `val.sort` be renamed to `values` in this function.